README.md

```diff
@@ -82,15 +82,21 @@ Just inject the service by type-hinting an argument with the `Jupi\PrestaShopWeb
 // src/Controller/ProductController.php
 namespace App\Controller;
 
-use Symfony\Component\HttpFoundation\Response;
 use Jupi\PrestaShopWebserviceBundle\Services\PrestaShopWebservice;
 // or
-use Jupi\PrestaShopWebserviceBundle\Services\PrestaShopWebserviceExtra;
+use Jupi\PrestaShopWebserviceExtra\PrestaShopWebserviceExtra;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
 
-class ProductController
+class ProductController extends AbstractController
 {
     /**
      * @Route("/products")
+     *
+     * @param PrestaShopWebservice $psWebservice
+     *
+     * @return Response
```

Comment:
I don't think this is needed, we already have param and return types in native PHP.
And this add too much lines imo, I would prefer to keep the example as small as possible.

Reply:
Also, if you want to make the change. I was thinking to convert the multiline phpdoc `@Route` in a single line PHP annotation like this: `#[Route('/products', name: 'products')]`

Reply:
Ok I agree!